bruno/collections/Rafiki/Rafiki Admin APIs/Get Wallet Address.bru

```diff
@@ -45,27 +45,34 @@ body:graphql {
           }
           walletAddressKeys {
             edges {
-                node {
-                    id
-                    jwk {
-                        kid
-                        x
-                        alg
-                        kty
-                        crv
-                    }
-                    revoked
-                    createdAt
+              node {
+                id
+                jwk {
+                  kid
+                  x
+                  alg
+                  kty
+                  crv
                 }
+                revoked
+                createdAt
+              }
             }
             pageInfo {
-                endCursor
-                hasNextPage
-                hasPreviousPage
-                startCursor
+              endCursor
+              hasNextPage
+              hasPreviousPage
+              startCursor
             }
           }
           status
+      	  additionalProperties {
+            properties {
+              key
+              value
+              visibleInOpenPayments
+            }
+          }
```

Comment:
can we get rid of the properties layer? just making `additionalProperties` the array of `{ key, value, visibleInOpenPayments }` ?

Reply:
@BlairCurrey sorted already.